src/chapters/jdbc-auth.xml

```diff
@@ -299,6 +321,60 @@ Type "help" for help.
                     to run any database upgrade scripts.</para>
             </important>
         </section>
+        <section xml:id="jdbc-auth-sqlserver">
+            <title>SQL Server</title>
+            <para>If using SQL Server, the database and schema must be created first.  The example below assumes
+                that you are running SQL Server on Linux, using the command-line tools to manage it.</para>
+            <informalexample>
+                <screen><prompt>$</prompt> <userinput>/opt/mssql-tools/bin/sqlcmd -S localhost -U SA</userinput>
+<prompt>Password:</prompt> <userinput><replaceable>password</replaceable></userinput>
+<prompt>1></prompt> <userinput>CREATE DATABASE <replaceable>guacamole_db</replaceable>;</userinput>
+<prompt>2></prompt> <userinput>GO</userinput>
+<prompt>1></prompt> <userinput>CREATE LOGIN <replaceable>guacamole_user</replaceable> WITH PASSWORD = '<replaceable>S0me_Password</replaceable>';</userinput>
```

Comment:
Any reason to avoid using the same password placeholder as the MySQL and PostgreSQL docs?

Reply:
I guess if it's a placeholder it doesn't matter, since we want people to change it from the default placeholder.  SQL Server does not allow passwords as simple as the placeholder, which is why I had tweaked it.  But I'll put it back.

Reply:
Fixed.